### ISSUE TITLE
[UX] File Field UI redesign.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1445]
Update: Plans in Drupal-land have shifted towards [Create accessible markup for a drag & drop file upload widget (and ensure there is an accessible alternative interaction)](https://www.drupal.org/project/drupal/issues/3064084) and the original issue slated for D8 has been closed in the meantime.

---

### Original request

This is a follow-up to #1380, the respective issue for https://www.drupal.org/node/2113931 and part of #779

Here's what they ~are~ were aiming for in D8:

![d8-file_field_redesign](https://cloud.githubusercontent.com/assets/2423362/11933698/655cae72-a851-11e5-9a41-15c07580d4c3.png)

**Comment by wesruv** [comment]
#want

**Comment by gifad** [comment]
Looking at related #1380, I discovered the File Resumable Upload project.
Made a quick port to Backdrop (the code a very clean and easy to figure out), and it works really fine !
You can pick it at https://github.com/gifad/backdrop-mediakit/tree/master/file_resup
Enjoy !

**Comment by BWPanda** [comment]
Ooh, so pretty! That'd be really nice to have in Backdrop. +1 from me!

**Comment by laryn** [comment]
Agreed! (By the screenshots, it looks like it includes this functionality, too: https://github.com/backdrop/backdrop-issues/issues/3297 )

**Comment by klonos** [comment]
I've updated the issue summary to mention that Drupal efforts have in the meantime shifted to https://www.drupal.org/project/drupal/issues/3064084 instead.
